system/lib/wasmfs/proxied_file_backend.cpp

```diff
@@ -54,10 +54,15 @@ class ProxiedFile : public DataFile {
   // A file with the chosen destination backend is created on a thread via
   // the ProxiedFile's proxy.
   ProxiedFile(mode_t mode,
-              backend_t backend,
-              std::shared_ptr<DataFile> file,
+              backend_t proxiedBackend,
+              backend_t baseBackend,
```

Comment:
`proxiedBackend` sounds like the backend that is being proxied to rather than the one that is being proxied. How about naming these `proxyBackend` and `proxiedBackend` or `proxyBackend` and `underlyingBackend` instead?

Not that `ProxiedFile` and `ProxiedBackend` seem fine, since from the user's point of view it's really the underlying, proxied files or backend that they care about.

Reply:
Sgtm. Done!